src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-details/rgw-multisite-details.component.html

```diff
@@ -1,123 +1,121 @@
+                    default
+                  </span>
+                  <span class="badge badge-warning me-2" *ngIf="node.data.is_master">
+                    master
+                  </span>
+                  <span class="badge badge-warning me-2" *ngIf="node.data.secondary_zone">
+                    secondary-zone
+                  </span>
+                  <div class="btn-group align-inline-btns" *ngIf="node.isFocused" role="group">
+                    <div [title]="editTitle" i18n-title>
+                      <button type="button" class="btn btn-light dropdown-toggle-split ms-1"
+                        (click)="openModal(node, true)" [disabled]="getDisable() || node.data.secondary_zone">
+                        <i [ngClass]="[icons.edit]"></i>
+                      </button>
+                    </div>
+                    <div [title]="deleteTitle" i18n-title>
+                      <button type="button" class="btn btn-light ms-1"
+                        [disabled]="isDeleteDisabled(node) || node.data.secondary_zone" (click)="delete(node)">
+                        <i [ngClass]="[icons.destroy]"></i>
+                      </button>
+                    </div>
                   </div>
-                </div>
-              </ng-template>
-            </tree-root>
-          </div>
-          <div class="col-sm-6 col-lg-6 metadata"
-               *ngIf="metadata">
-            <legend>{{ metadataTitle }}</legend>
-            <div>
-              <cd-table-key-value cdTableDetail
-                                  [data]="metadata">
-              </cd-table-key-value>
+                </ng-template>
+              </tree-root>
+            </div>
+            <div class="col-sm-6 col-lg-6 metadata" *ngIf="metadata">
+              <legend>{{ metadataTitle }}</legend>
+              <div>
+                <cd-table-key-value cdTableDetail [data]="metadata">
+                </cd-table-key-value>
+              </div>
             </div>
           </div>
         </div>
       </div>
-    </div>
-  </div>
-</div>
+    </ng-template>
+  </ng-container>
+  <ng-container ngbNavItem="syncPolicy">
+    <a ngbNavLink i18n>Sync Policy</a>
+    <ng-template ngbNavContent>
+      <legend i18n>
+        Multisite Sync Policy
+        <cd-help-text>
+          Multisite bucket-granularity sync policy provides fine grained control of data movement between buckets in different zones.
+        </cd-help-text>
+      </legend>
+      <cd-table #table
+                [data]="syncPolicyData"
+                [columns]="columns"
+                columnMode="flex"
+                selectionType="single"
+                [searchableObjects]="true"
+                [hasDetails]="true"
+                [serverSide]="false"
+                [count]="count"
+                [maxLimit]="25"
+                [toolHeader]="true">
+      </cd-table>
+    </ng-template>
+  </ng-container>
+</nav>
```

Comment:
let's create a new component for the sync policy because we are going to put a lot of informations here and its best to do it in a separate component of its own. probably under `frontend/src/app/ceph/rgw/`

Reply:
yeah, it make sense